packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -391,6 +391,10 @@ export default mixins(
 					}
 				}
 
+				if (this.parameter.type === 'multiOptions' && typeof returnValue === 'string' && !this.isValueExpression) {
+					returnValue = (returnValue || '').split(',');
```

Comment:
this works well. but we should do this as part of removing expression so that the change is reflected correctly in workflow object/store
before
<img width="1355" alt="Screenshot 2022-04-12 at 17 42 58" src="https://user-images.githubusercontent.com/4711238/163001687-5aa81b7d-c61d-4688-8963-d432ff2ed3b1.png">
after
<img width="1311" alt="Screenshot 2022-04-12 at 17 41 26" src="https://user-images.githubusercontent.com/4711238/163001749-50c7d42f-bd05-418b-946a-49ffdbd6c15f.png">